docs/operation/join-a-cs.md

```diff
@@ -236,21 +237,37 @@ You will see logs similar to the following once successfully created new miner-i
 
 ### damocles-manager configurations
 
-Tailor `~/.damocles-manager/sector-manager.cfg` to your own hardware. Detailed instructions on each configurations could be found [here](/zh/cluster/Venus-Sector-Manager).
+Tailor `~/.damocles-manager/sector-manager.cfg` to your own hardware. Detailed instructions on each configurations could be found [here](/zh/cluster/damocles-manager).
 
 ```toml
 [Common]
   [Common.API]
-    # *mandatory* node address (chain service)
-    Chain = "/ip4/{api_host}/tcp/{api_port}"
-    # *mandatory* messager address (chain service)
-    Messager = "/ip4/{api_host}/tcp/{api_port}"
-    # market node address
-    # Market = "/ip4/{api_host}/tcp/{api_port}"
-    # *mandatory* gateway address (chain service)
-    Gateway = "/ip4/{api_host}/tcp/{api_port}"
-    # *mandatory* authentication token (chain service)
-    Token = "{auth token}"
+  # Gateway service infos, required, string type
+  # Fill in according to the actual situation of the service used
+  # For each one contained, if the item is valid as a token-included-info-string ("{token}:{multiaddr}"), the token included would be used to construct the rpc client instead of the common token.
```

Comment:
```suggestion
  # If a token is supplied in the format of "{token}:{multiaddr}", then the token supplied would be used to construct the RPC client instead of the common token generated by sophon-auth from Token configuration item.
```